test/exometer_SUITE.erl

```diff
@@ -486,6 +488,37 @@ test_status(_Config) ->
      {ref, undefined}] = exometer:info(M1),
     ok.
 
+%% Ensure a slide ignores values which are outdated as per its configuration of
+%% time_span. This is important in cases with low update frequencies.
+test_slide_ignore_outdated(_Config) ->
+   M = [?MODULE, hist, ?LINE],
+
+   ok = exometer:new(
+          M, ad_hoc, [{module, exometer_histogram},
+                      {type, histogram},
+                      {histogram_module, exometer_slide},
+                      {time_span, 5}]),
+   % check that no entries exist
+   {ok, V1} = exometer:get_value(M),
+   0 = proplists:get_value(n, V1),
+
+   % add entry
+   ok = exometer:update(M, 1234),
+
+   % check that new entry exists
+   {ok, V2} = exometer:get_value(M),
+   ct:pal("~p", [V2]),
```

Comment:
```suggestion
   ct:log("New entry: ~p", [V2]),
```

Reply:
I removed it altogether, forgot to do so before. Thanks.